pkgs/applications/science/electronics/hyp2mat/default.nix

```
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchFromGitHub
+, libtool
+, gnumake
+, bison
+, flex
+, gengetopt
+, help2man
+, groff
+, libharu
+, autoreconfHook
+, pkgconfig
+, libpng
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hyp2mat";
+  version = "0.0.17";
+
+  src = fetchFromGitHub {
+    owner = "koendv";
+    repo = "hyp2mat";
+    rev = version;
+    sha256 = "04bd2vrn8y1izir7pwl34a60cg2v05baqcrmpfdx5fvf33bbflaj";
+  };
+
+  buildInputs = [
+    libharu
+    libpng
+    zlib
+    autoreconfHook
+    pkgconfig
+    libtool
+    gnumake
+    bison
+    flex
+    gengetopt
+    help2man
+    groff
+  ];
```

commands that need to be executed at build-time should go in nativeBuildInputs
```suggestion
  nativeBuildInputs = [
      autoreconfHook
      pkgconfig
  ];

  buildInputs = [
    libharu
    libpng
    zlib


    bison
    flex
    gengetopt
    help2man
    groff
  ];
```

gnumake is available by default, and libtool is brought in by `autoreconfhook`

---

Done.